migration/idempotent/001-base.sql

```diff
@@ -1995,7 +2062,8 @@ LANGUAGE PLPGSQL STABLE;
 GRANT EXECUTE ON FUNCTION _prom_catalog.get_cagg_info(text, text) TO prom_reader;
 
 CREATE OR REPLACE FUNCTION prom_api.is_stale_marker(value double precision)
-RETURNS BOOLEAN
+    RETURNS BOOLEAN
+    SET search_path = pg_catalog
```

Comment:
this should def by inlinable so let's remove this and just schema qualify the pg_catalog.float8send

Reply:
done